tools/integration_tests/setup/setup.go

```diff
@@ -0,0 +1,154 @@
+package setup
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path"
+
+	"github.com/googlecloudplatform/gcsfuse/tools/util"
+)
+
+var testBucket = flag.String("testbucket", "", "The GCS bucket used for the test.")
+var mountedDirectory = flag.String("mountedDirectory", "", "The GCSFuse mounted directory used for the test.")
+
+var (
+	binFile string
+	logFile string
+	testDir string
+	tmpDir  string
+	mntDir  string
+)
+
+func TestBucket() string {
+	return *testBucket
+}
+
+func MountedDirectory() string {
+	return *mountedDirectory
+}
+
+func SetLogFile(logFileValue string) {
+	logFile = logFileValue
+}
+
+func LogFile() string {
+	return logFile
+}
+
+func SetBinFile(binFileValue string) {
+	binFile = binFileValue
+}
+
+func BinFile() string {
+	return binFile
+}
+
+func SetTestDir(testDirValue string) {
+	testDir = testDirValue
+}
+
+func TestDir() string {
+	return testDir
+}
+
+func SetTmpDir(tmpDirValue string) {
+	tmpDir = tmpDirValue
+}
+
+func TmpDir() string {
+	return tmpDir
+}
+
+func SetMntDir(mntDirValue string) {
+	mntDir = mntDirValue
+}
+
+func MntDir() string {
+	return mntDir
+}
+
+func SetUpTestDir() error {
+	var err error
+	testDir, err := os.MkdirTemp("", "gcsfuse_readwrite_test_")
+	if err != nil {
+		return fmt.Errorf("TempDir: %w\n", err)
+	}
+	SetTestDir(testDir)
+
+	err = util.BuildGcsfuse(TestDir())
+	if err != nil {
+		return fmt.Errorf("BuildGcsfuse(%q): %w\n", TestDir(), err)
+	}
+
+	SetBinFile(path.Join(TestDir(), "bin/gcsfuse"))
```

Comment:
setter and getter are for other packages to access these variable. not for this package and more importantly this file

Reply:
fixed